notebook_pge_wrapper/cli.py

```diff
@@ -56,9 +64,26 @@ def create(project):
     if not os.path.exists(notebook_pges_directory):
         os.mkdir(notebook_pges_directory)
 
+    # create sample pge notebook
+    copyfile(sample_pge_notebook_file, os.path.join(notebook_pges_directory, __SAMPLE_PGE_NOTEBOOK_FILE))
+
     # create README.md
     copyfile(readme_file, os.path.join(project, __README_FILE))
 
+    # create pge_create notebook
+    pge_create_notebook_dest = os.path.join(project, __PGE_CREATE_NOTEBOOK_FILE)
+    with open(pge_create_notebook_file, 'r') as pge_create_infile, \
```

Comment:
we can rename `pge_create_infile` to `fin` and `pge_create_outfile` to `fout` for readability